Makefile

```diff
@@ -1,37 +1,41 @@
-TARGETS = naive cuda cupla-cuda-async cupla-seq-seq-async cupla-seq-seq-sync cupla-tbb-seq-async cupla-omp2-seq-async kokkos-serial kokkos-openmp kokkos-cuda
+TARGETS = naive cuda cupla kokkos
 
 .PHONY: all debug clean $(TARGETS)
 
-CXX := g++-8
+CXX := g++
 CXX_FLAGS := -O2 -std=c++14 -ftemplate-depth-512
 CXX_DEBUG := -g
 
-NVCC := /usr/local/cuda-10.1/bin/nvcc -ccbin $(CXX)
-NVCC_FLAGS := -O2 -std=c++14 --expt-relaxed-constexpr -w --generate-code arch=compute_50,code=sm_50
+NVCC := /usr/local/cuda/bin/nvcc -ccbin $(CXX)
+NVCC_FLAGS := -O2 -std=c++14 --expt-relaxed-constexpr -w --generate-code arch=compute_35,code=sm_35
 NVCC_DEBUG := -g -lineinfo
 
-ALPAKA_BASE := $(HOME)/src/alpaka/alpaka
-CUPLA_BASE  := $(HOME)/src/alpaka/cupla
+ALPAKA_BASE := /usr/local/alpaka/alpaka
+CUPLA_BASE  := /usr/local/alpaka/cupla
 CUPLA_FLAGS := -DALPAKA_DEBUG=0 -I$(ALPAKA_BASE)/include -I$(CUPLA_BASE)/include
 
+RED := '\033[31m'
+RESET := '\033[0m'
+
 all: $(TARGETS)
 
 # Recommended to include only after the first target
 # https://github.com/kokkos/kokkos/wiki/Compiling#42-using-kokkos-makefile-system
 ifdef KOKKOS_BASE
   include $(KOKKOS_BASE)/Makefile.kokkos
-  CXX_KOKKOS := $(CXX)
   ifeq ($(KOKKOS_INTERNAL_USE_CUDA),1)
     CXX_KOKKOS := $(NVCC_WRAPPER)
     KOKKOS_CXXFLAGS += --expt-relaxed-constexpr
+  else
+    CXX_KOKKOS := $(CXX)
   endif
 endif
 
 
 debug: $(TARGETS:%=debug-%)
```

Comment:
By eye the mechanism for debug builds would need an update as well (e.g. there is no `debug-cupla` or `debug-kokkos` target).